test/js/scp-api.test.js

```diff
@@ -55,8 +51,28 @@ describe('JavaScript client for SCP REST API', () => {
       })
     )
     process.nextTick(() => {
-      jest.restoreAllMocks()
       done()
     })
   })
+
+  it('catches 500 errors', async () => {
+    const mockErrorResponse = {
+      type: 'basic',
+      url: 'http://localhost:3000/single_cell/api/v1/search?type=study',
+      redirected: false,
+      status: 500,
+      ok: false,
+      statusText: 'Internal Server Error'
+    }
+    jest.spyOn(global, 'fetch').mockReturnValue(Promise.resolve(mockErrorResponse))
+    const actualResponse = await scpApi('/test/path', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json'
+      }
```

Comment:
Does this test still pass if you omit these lines?

```suggestion
```

I believe these fetch `init` properties are automatically supplied by [`defaultInit` in `scp-api.js`](https://github.com/broadinstitute/single_cell_portal_core/blob/master/app/javascript/lib/scp-api.js#L23), but I could be missing something.